src/multi_stage/endogenous_retirement.jl

```diff
@@ -22,6 +22,56 @@ function get_retirement_stage(cur_stage::Int, lifetime::Int, stage_lens::Array{I
     return Int(ret_stage)
 end
 
+function compute_min_cumulative_retirements!(t::Int,inputs_d::Dict)
```

Comment:
To make the code easier to read, consider adding a simple one-line function like

`dfgen(stage) = inputs_d[stage]["dfGen"]`

and also 

```
dfvrestore(stage) = inputs_d[stage]["dfVRE_STOR"]
```

This also has the advantage that it makes the interface smaller. If you ever want to change `"dfVRE_STOR"` to another key it's easier since it's only in one place.

Reply:
Could also do something like

```
function xxx(a,b,c):
    if !isempty(a)
        inputs[...][!, Symbol(b)] = sum( .... Symbol(c) for p in 1:t)
     end
     
table = [("VS_DC", "Min_Cum_Retired_Cap_Inverter_MW", "Min_Retired_Cap_Inverter_MW"),
              (..., ..., ...),
              ]
for row in table:
    a,b,c = row
    xxx(a,b,c)
end
```

Reply:
Two more suggestions to improve readability in addition to @cfe316 comments:

- You could use a `Symbol` to select a column (e.g.,`:Min_Cum_Retired_Energy_Cap_MW`) instead of converting a `String` into a `Symbol` every time, (e.g., `Symbol("Min_Cum_Retired_Energy_Cap_MW")`)

- The usual convention for mutating functions (e.g., `myfun!(t::Int,inputs_d::Dict)` is that the argument that is modified is placed first (Ref.: https://docs.julialang.org/en/v1/manual/style-guide/#Write-functions-with-argument-ordering-similar-to-Julia-Base)
```suggestion
function compute_min_cumulative_retirements!(inputs_d::Dict, t::Int)
```